packages/pyroscope-flamegraph/src/Toolbar.tsx

```diff
@@ -175,24 +176,22 @@ const Toolbar = memo(
             selectedNode={selectedNode}
             onFocusOnSubtree={onFocusOnSubtree}
           />
+          <Divider />
         </>
       ),
-      width: TOOLBAR_SQUARE_WIDTH,
+      width: TOOLBAR_SQUARE_WIDTH + DIVIDER_WIDTH,
     };
     const viewSectionItem = enableChangingDisplay
       ? {
           el: (
-            <>
-              <Divider />
-              <ViewSection
-                flamegraphType={flamegraphType}
-                view={view}
-                updateView={updateView}
-              />
-            </>
+            <ViewSection
+              flamegraphType={flamegraphType}
+              view={view}
+              updateView={updateView}
+            />
           ),
           // sandwich view is hidden in diff view
-          width: TOOLBAR_SQUARE_WIDTH * (flamegraphType === 'single' ? 4 : 3),
+          width: TOOLBAR_SQUARE_WIDTH * (flamegraphType === 'single' ? 4 : 3), // 1px is to display divider
```

Comment:
Just seeing this now, but it feels flaky...Ideally I would have 2 sets of `const ToolbarItems = ToolbarItem[]`, one for `single` and another for `double`, then calculations would be derived from that.

(You don't have to act on it, I am just thinking out loud)